blog/electron-22-0.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Electron 22.0.0
+date: 2022-11-22T00:00:00.000Z
+authors:
+    - name: vertedinde
+      url: 'https://github.com/vertedinde'
+      image_url: 'https://github.com/vertedinde.png?size=96'
+    - name: georgexu99
+      url: 'https://github.com/georgexu99'
+      image_url: 'https://github.com/georgexu99.png?size=96'
+slug: electron-22-0
+
+---
+
+Electron 22.0.0 has been released! It includes upgrades to Chromium `108`, V8 `10.8`, and Node.js `16.17.1`. Read below for more details!
```

Comment:
If we want to make this the "most important" or "above the fold" part of the release blog, it would make more sense for us to highlight the major new additions (i.e. UtilityProcess) and major changes (Windows version deprecations) instead of the stack upgrades.

I think these are a lot more interesting to blog readers and will catch their attention a lot better.